describe/describe.py

```diff
@@ -2,133 +2,244 @@
 
 import warnings
 
+import mlrun.artifacts
```

Comment:
Why do you import this one? anyway when importing an entire sub-package it is best to give it an alias:
```
import torch.nn as nn
```
Otherwise, simply import mlrun.